x/auth/types/autocli.go

```diff
@@ -0,0 +1,26 @@
+package types
+
+import autocliv1 "cosmossdk.io/api/cosmos/autocli/v1"
+
+var AutoCLIOptions = &autocliv1.ModuleOptions{
+	Query: &autocliv1.ServiceCommandDescriptor{
```

Comment:
thinking out loud: a lot of this can be defined declaratively. Probably the only thing that cannot are custom query/tx commands. Is it worth adding this to the ModuleConfig? Or are the ModuleConfigs committed to state?

Reply:
I was thinking a new field here: https://github.com/cosmos/cosmos-sdk/blob/main/proto/cosmos/auth/module/v1/module.proto#L8 (and in each module which wants to leverage this)

Reply:
I think that gets us back into the whole debate over config. Probably not worth it